contracts/InternalOptimisticOracle.sol

```diff
@@ -47,8 +63,15 @@ contract InternalOptimisticOracle is Testable {
         oo = OptimisticOracleV2Interface(finder.getImplementationAddress(OracleInterfaces.OptimisticOracleV2));
     }
 
-    mapping(bytes32 => Request) public requests;
-
+    /**
+     * @notice Requests a price from the Internal Optimistic Oracle.
+     * @param timestamp timestamp of the price being requested.
+     * @param ancillaryData ancillary data of the price being requested.
+     * @param reward reward offered to a successful proposer. Will be pulled from the caller. Note: this can be 0, which could
+     * make sense if the contract requests and proposes the value in the same call or provides its own reward system.
+     * @param bond custom proposal bond to set for request. If set to 0, defaults to the final fee.
```

Comment:
Does it really set bond to final fee if 0? `oo.setBond` in `disputePrice` would explicitly set it to 0

Reply:
We always call [oo.setBond](https://github.com/UMAprotocol/dev-quickstart/blob/c5d8a9148546e641d289cafd06e80bd8f434c35e/contracts/InternalOptimisticOracle.sol#L161) in the dispute, so if we pass bond=0, both `oo.proposePriceFo` and `oo.disputePriceFor` will pull a `finalFee` amount of currency each, `2*finalFee` in total

How would you phrase it?